tests/test_admin.py

```diff
@@ -92,3 +93,67 @@ def test_admin(app):
             data={'rowid': -42, 'action': 'activate'},
             follow_redirects=True
         )
+
+
+def test_admin_createuser(app):
+    """Test flask-admin user creation"""
+    assert isinstance(user_adminview, dict)
```

Comment:
consider creating `admin` fixture in `conftest.py` from lines 100-110 (and 44-56)